modules/pekko/docs/README.md

```diff
@@ -0,0 +1,35 @@
+# Pekko module for PureConfig
+
+Adds support for selected [Pekko](https://pekko.apache.org/) classes to PureConfig.
+
+## Add pureconfig-pekko to your project
+
+In addition to [core PureConfig](https://github.com/pureconfig/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-pekko" % "@VERSION@"
```

Comment:
Running `pekko-http-docs/mdoc` should have this replaced.

```suggestion
libraryDependencies += "com.github.pureconfig" %% "pureconfig-pekko" % "0.17.5"
```

Reply:
Same as above.
Regenerated the files in 8655d5b